.docs/content/1.installation/1.linux/2.verify-installation.md

```diff
@@ -0,0 +1,88 @@
+# Verify Installation
+
+After [installing ArmoniK](./1.installation.md), it is time to execute some simple test to check that everything is working as expected. In order to do that, the ArmoniK team is providing three simple tests to verify that the deployment wen well.
+
+## Seq
+
+First of all, you can try to connect to the log server [Seq](https://datalust.co/) to check that it is working correctly.
+
+You can find the Seq URL printed on your console after the deployment.
+
+::alert{type="info"}
+You can also retrieve the `seq.web_url` from the Terraform outputs `monitoring/generated/monitoring-output.json`. The default port is `8080` but the ip address can be different depending on your machine.
+::
+
+Example:
+
+```bash
+http://<ip_address>:8080
+```
+
+<!-- TODO: need to be confirmed -->
+No credentials are required to connect to Seq.
```

Comment:
Only true when the ingress is not deployed OR if it is deployed without mTLS. TLS will show an insecure connection but no problem to connect. With mTLS the user must add the client certificates to their browser

Reply:
Yes, a section about authentification is missing. I don't know how we can talk about it here. Maybe juste a warning at the beginning to explain what the user will deploy?

Reply:
Will add some warning.